sample_apps/php/README.md

```diff
@@ -0,0 +1,29 @@
+# Getting started with Amazon Timestream with PHP
+
+The example PHP application gives developers a basic understanding of how to begin using Amazon Timestream with PHP. The application exemplifies the beginning phases of database creation with some sample data population and how to execute queries against a database.
+
+-----
+## How to use
+### Installation
+#### PHP Installation
+Navigate to [PHP official website](https://www.php.net/downloads.php) and install the stable version of PHP
```

Comment:
SPecify version the sample app has been tested with

Reply:
Done, system information has been added